doc/openapi/v0/cyphernode-api.yaml

```diff
@@ -350,6 +350,135 @@ paths:
                 $ref: '#/components/schemas/ApiResponseTemporarilyUnavailable'
+  /get_txns_by_watchlabel/{label}/{count}:
+    get:
+      parameters:
+        - in: "path"
+          name: "label"
+          description: "Xpub label"
+          required: true
+          schema:
+            type: "string"
+        - in: "path"
+          name: "count"
+          description: "Number of transactions to return"
+          required: false
+          schema:
+            type: "number"
+      tags:
+        - "watching transactions"
+        - "core features"
+      summary: "Get list of transactions observed for addreses belonging to this label"
+      description: "Get list of transactions observed for addreses belonging to this label"
+      operationId: "get_txns_by_watchlabel"
+      responses:
+        '200':
+          description: "successful operation"
+          content:
+            application/json:
+              schema:
+                type: "object"
+                required:
+                  - "label_txns"
+                properties:
+                  watches:
+                    type: "array"
+                    items:
+                      $ref: '#/components/schemas/WatchXPubTxn'
+        '403':
+          $ref: '#/components/schemas/ApiResponseNotAllowed'
+        '503':
+          description: "Resource temporarily unavailable"
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ApiResponseTemporarilyUnavailable'
+  /get_txns_spending/{count}/{skip}:
+    get:
+      parameters:
+        - in: "path"
+          name: "count"
+          description: "Number of txns to return"
+          required: false
+          schema:
+            type: "number"
+        - in: "path"
+          name: "skip"
+          description: "Skip/offset to start from"
+          required: false
+          schema:
+            type: "number"
+      tags:
+        - "spending transactions"
+        - "core features"
+      summary: "Get list of spending wallets transactions"
+      description: "Get list of spending wallets transactions"
+      operationId: "get_txns_spending"
+      responses:
+        '200':
+          description: "successful operation"
+          content:
+            application/json:
+              schema:
+                type: "object"
+                required:
+                  - "txns"
+                properties:
+                  watches:
+                    type: "array"
+                    items:
+                      $ref: '#/components/schemas/TransactionsSpending'
+        '403':
+          $ref: '#/components/schemas/ApiResponseNotAllowed'
+        '503':
+          description: "Resource temporarily unavailable"
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ApiResponseTemporarilyUnavailable'
+  /get_unused_addresses_by_watchlabel/{label}/{count}:
```

Comment:
see /get_txns_spending/{count}/{skip}

For me count would be optional and label required, so count should move into the query string

Reply:
same as previous comment.